src/EnergyPlus/General.cc

```diff
@@ -421,44 +421,22 @@ std::string &strip_trailing_zeros(std::string &InputString)
     return InputString; // Allows chaining
 }
 
-void MovingAvg(Array1A<Real64> const DataIn, // input data that needs smoothing
-               int const NumDataItems,       // number of values in DataIn
-               int const NumItemsInAvg,      // number of items in the averaging window
-               Array1A<Real64> SmoothedData  // output data after smoothing
-)
+void MovingAvg(Array1D<Real64> &DataIn, int const NumItemsInAvg)
```

Comment:
This function was relying on a temp array to calculate the smooted array. The change acts directly on the referenced array and gets rid of 2 arguments.